src/rez/bind/_pymodule.py

```diff
@@ -34,8 +34,14 @@ def copy_module(name, destpath):
          "print(%s.__path__[0] if hasattr(%s, '__path__') else '')" % (name, name)])
 
     if out:
-        srcpath = out
-        shutil.copytree(srcpath, os.path.join(destpath, name))
+        if name == "pip":
+            path = subprocess.check_output('python -c "import pip; print(pip.__path__)"', shell=True)
+            # retrieve second line since first is the Python 2 deprecation warning
```

Comment:
This seems too fragile, and likely to break after the py3 port. I'd suggest:
* see if using shell=True across the board works. If so do that, rather than special-casing pip;
* see if simply taking the last line of stdout works (rather than assuming there are >1 lines and that line[0] is a deprecation warning). Again this would remove the need for special-casing for pip.

Reply:
Hello Allan sounds good to me, 

> see if using shell=True across the board works. If so do that, rather than special-casing pip;

This is what I originally had in mind but the [warning](https://docs.python.org/2/library/subprocess.html) about using shell=True on the documentation website and many other [posts](https://stackoverflow.com/a/3172488) made me a bit sceptical. I will do the proposed fixes and report back shortly. 

Maybe is worth adding shell=True across the board only if Windows is detected?

Reply:
@nerdvegas Using shell=True across the board still works as expected on Windows. Saying that I noticed a significant time increase during the binding Python part on Linux. I believe it might be worth adding the platform check and enabling the flag only on Windows.

Reply:
I think it's slow, because on Linux arguments can't be passed as `list` when used with `shell=True`. You can `" ".join(args)` prior to passing it, on either platform, and that should fix it. On Windows, shell also exposes the local PATH, which `shell=False` doesn't, which has a slightly different behavior than Linux.

Reply:
Thank you Marcus, seems to be alright now. I have settled for a platform check instead I believe is cleaner that way. shell=True is only ever required on Windows because of the different behaviour you describe above.

Eventually we will need to refactor the code a bit to use the new [subprocess.run](https://docs.python.org/3.5/library/subprocess.html#subprocess.run) instead of the old Popen.

Reply:
@mottosso If you can give it a spin by the way and let me know if it works it would be great! It works on my Windows 10 VM. Should work without admin rights console.

Reply:
I currently doesn't work for me, as I'm on Python 3. Personally I wouldn't rely on the deprecation message being consistent enough to rely on in this way. How about printing something additional alongside the path, something to identify the output as yours? E.g:

```py
print("my_unique_id: %s" % pip.__path__)
```

And then..

```py
out.split("my_unique_id: ", 1)
```

Reply:
@mottosso Hello Marcus, please try the latest version. The code has changed. Thank you.

Reply:
That looks good to me. If you add `universal_newlines=True` you'll also get consistent output between Python 2 and 3, but it shouldn't affect the behaviour under Python 2. 👍